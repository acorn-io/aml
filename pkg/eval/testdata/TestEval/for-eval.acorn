args: scale: 1
for i in std.range(1,args.scale+1) {
	containers: "a\(i)": build: "./foo"
	acorns: "ac\(i)": build: "./afoo"
	services: "sc\(i)": build: "./afoo"
}