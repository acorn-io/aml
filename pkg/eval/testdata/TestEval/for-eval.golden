{
  "acorns": {
    "ac1": {
      "build": "./afoo"
    }
  },
  "containers": {
    "a1": {
      "build": "./foo"
    }
  },
  "services": {
    "sc1": {
      "build": "./afoo"
    }
  }
}